README.md

```diff
@@ -1,2 +1,24 @@
 # e2e-nativek8s
-OpenEBS E2E Pipelines
+
+This repository's `release-branch` branch contains plateform execution scripts for each indivisual test which is meant to run for inducing chaos and testing zfs-localpv components. It includes several stages to perform different scenraios in end to end work-flow. To see the README's and test results of pipeline switch to the `release-branch` of this repository.
+
+To see the CI/E2E summary of the gitlab pipelines for e2e-nativek8s project visit `openebs.ci` and switch to `k8s` tab under `stable releases` category.
+
+## Pipeline stages for running against release-branch
+
+1. cluster-setup
+2. provisioner
+3. functional
+4. chaos
```

Comment:
chaos stage is not there in this pipeline

Reply:
As of now no test cases are there in chaos stage, but kept there for future reference